src/tigerbeetle.zig

```diff
@@ -400,8 +400,8 @@ pub const CreateTransferResult = enum(u32) {
         assert(set.count() == set.capacity());
     }
 
-    /// TODO(zig): This enum is be ordered by precedence, but it crashes
```

Comment:
s/"This enum"/CreateTransferResult to clarify that we aren't referring to `Ordered`.